store/contentSchemas.json

```diff
@@ -95,6 +95,25 @@
           "$ref": "app:vtex.native-types#/definitions/text"
         }
       }
+    },
+    "Image": { 
+      "properties": {
+        "src": {
+          "title": "admin/editor.image.src.title",
+          "$ref": "app:vtex.native-types#/definitions/url",
```

Comment:
I think here you could use image native type:

https://github.com/vtex-apps/native-types/blob/master/pages/contentSchemas.json#L27-L50

But this would lead to a breaking change, so the component should support the old prop `src` and `alt` as well.

Reply:
Can I open a different PR for this one?